### PR TITLE
feat(p-search): add autocomplete function to PSearch component

### PR DESCRIPTION
**Pull request by sulmoJ**
### To Reviewers
- [ ] Skipping review is not a problem (```style```, ```chore``` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
  - [ ] These changes require a usage change
- [ ] Others (performance improvement, refactoring, CI/CD, etc.)

### Checklist
- [x] Updated Storybook documents
- [x] Tested with console

### Description

- Added auto-completion.

https://user-images.githubusercontent.com/50662170/197997851-ae4aa7ad-b3a3-431b-aae4-5908c340e4de.mov

### CODE REVIEW

File: src/inputs/search/search/PSearch.vue [hunk 1]
```diff
@@ -1,52 +1,73 @@
...
+             :class="{ focused: proxyIsFocused, invalid, disabled }"
+        >
+            <p-i v-if="!disableIcon && !proxyIsFocused && !value && !readonly" class="left-icon" name="ic_search"
+                 color="inherit"
+            />
+            <slot name="left" v-bind="{ value, placeholder: placeholderText }" />
+            <span class="input-wrapper">
+                <slot name="default" v-bind="{ value, placeholder: placeholderText }">
+                    <input v-focus.lazy="proxyIsFocused"
+                           v-bind="$attrs"
```
Comment (WANZARGEN) [inline src/inputs/search/search/PSearch.vue:14]: 👍👍👍

File: src/inputs/search/search/PSearch.vue [hunk 2]
```diff
@@ -82,22 +103,125 @@ export default defineComponent<SearchProps>({
...
+        } = useContextMenuFixedStyle({
+            useFixedMenuStyle: computed(() => props.useFixedMenuStyle),
+            visibleMenu: computed(() => props.visibleMenu),
+        });
+        const contextMenuFixedStyleState = reactive({
+            proxyVisibleMenu, targetRef, targetElement, contextMenuStyle,
+        });
         const state = reactive({
-            proxyIsFocused: makeOptionalProxy('isFocused', vm, false),
+            proxyIsFocused: useProxyValue('isFocused', props, emit),
```
Comment (WANZARGEN) [inline src/inputs/search/search/PSearch.vue:149]: nice refactoring :)

File: src/inputs/search/search/PSearch.vue [hunk 3]
```diff
@@ -82,22 +103,125 @@ export default defineComponent<SearchProps>({
...
+            loading: useProxyValue('loading', props, emit),
```
Comment (WANZARGEN) [inline src/inputs/search/search/PSearch.vue]: I have a question!
The 'loading' state is declared with 'useProxyValue' hook, which means that the 'update:loading' event would be emitted.
1. I wonder why 🤔
2. If there is proper reason, I think 'update:loading' arg type should be declared in `story-helper.ts`.

**Review by WANZARGEN — APPROVED** [review]
Overall, looks good to me :)
I left one question!

File: src/inputs/search/search/PSearch.vue (hunk 3 above)
Comment (sulmoJ) [reply src/inputs/search/search/PSearch.vue]: 1. Added to manipulate the loading of contenxt-menu when disableHandler is true.
2. thx!!

(Invalid variable name(loading) has been additionally modified.)

**Pull request merged**